docs/src/appendix/snfoundry-toml.md

```diff
@@ -0,0 +1,100 @@
+# The Manifest Format
+
+The `snfoundry.toml` contains the project's manifest and allows specifying sncast settings.
+You can configure sncast settings and place arguments in it to not have to provide them in CLI along with the commands.
+If `snfoundry.toml` is not found in the root directory, sncast will look in all parent directories. 
+If it is not found, default values will be used.
+
+## `snfoundry.toml` contents
+
+### `[sncast.<profile-name>]`
+
+
+```toml
+[sncast.myprofile]
+url = "http://127.0.0.1:5055/"
+account = "user1"
+```
+
+All fields are optional and do not have to be provided. In case a field is not defined in a manifest file, it must be provided in CLI when executing a relevant `sncast` command.
```

Comment:
I think it'd be nice to link to profiles explanation or briefly explain it here.

Reply:
Added